reporter/src/main/kotlin/reporters/WebAppReporter.kt

```diff
@@ -22,6 +22,7 @@ package com.here.ort.reporter.reporters
 import com.here.ort.model.jsonMapper
```

Comment:
This is now unused and must be removed to avoid a *detekt*  issue.

Reply:
Done